src/components/SocialLinks-Color/index.js

```diff
@@ -37,6 +37,7 @@ const SocialLinksColor = () => {
                         href="https://github.com/layer5io"
                         target="_blank"
                         rel="noreferrer"
+                        className="github-anchor"
```

Comment:
Is this class being used?

Reply:
No, not actually, I was planning to use it but it wasn't required in the end, I am removing it

Reply:
I have removed it,sir!